torch/_functorch/config.py

```diff
@@ -162,6 +162,10 @@ def remote_autograd_cache_default() -> Optional[bool]:
 # tokens.
 unlift_effect_tokens = False
 
+
+# Run aot eager decomp partition with CrossRefFakeMode
+fake_tensor_crossref = False
```

Comment:
to check, your PR made this both a config option and a backend?

Reply:
that's correct